code/__DEFINES/materials.dm

```diff
@@ -4,6 +4,9 @@
 /// Hard materials, such as iron or metal
 #define MAT_CATEGORY_RIGID "rigid material"
 
+/// Materials that can be used to craft items
+#define MAT_CATEGORY_ITEM_MATERIAL "item material"
```

Comment:
Was there an issue with using `MAT_CATEGORY_RIGID` for this?

Reply:
no but i dont want hauntium to be used for turfs.